<launch>
  <arg name="global_path_topic" default="/global_path" />
  <arg name="goal_topic" default="/move_base_simple/goal" />
  <arg name="odom_topic" default="/odom" />
  <arg name="frame_id" default="/real_map" />
  <arg name="distance_step" default="1.0" />

  <node pkg="lattice_planner" type="simple_path_planner" name="simple_path_planner" output="screen" >
    <param name="global_path_topic" value="$(arg global_path_topic)"/>
    <param name="goal_topic" value="$(arg goal_topic)"/>
    <param name="odom_topic" value="$(arg odom_topic)"/>
    <param name="frame_id" value="$(arg frame_id)"/>
    <param name="distance_step" value="$(arg distance_step)" />
  </node>
</launch>